source/code/scxsystemlib/disk/statisticallogicaldiskenumeration.cpp

```diff
@@ -234,6 +234,12 @@ namespace SCXSystemLib
             SCXCoreLib::SCXHandle<StatisticalLogicalDiskInstance> disk = *iter;
             disk->Update();
 
+//Fix for Bug10019956
+#if defined(sun)
+        if(disk->m_fsType == L"zfs" && (disk->m_device).find(L"/") != -1 && (disk->m_device).find(L"/") != 0)
```

Comment:
Finally, perhaps in a comment here, or perhaps in comments in the code, can you clarify exactly why ZFS is counted wrong when other file systems are not. Perhaps this is even better for a multi-line commit comment, so we can see exactly WHY things were changed down the road if we need to (without pulling up the PR).